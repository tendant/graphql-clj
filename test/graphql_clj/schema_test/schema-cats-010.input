type Hello {
  world(things: [String]): String
}
