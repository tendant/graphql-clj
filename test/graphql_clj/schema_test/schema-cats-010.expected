{:objects
 {:Hello
  {:fields
   {:world {:args {"things" {:type (list String)}}, :type String}}}}}
